pubsub/src/main/java/com/github/twitch4j/pubsub/domain/UpdatedUnbanRequest.java

```diff
@@ -0,0 +1,28 @@
+package com.github.twitch4j.pubsub.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.Setter;
+import lombok.ToString;
+
+@Data
+@Setter(AccessLevel.PRIVATE)
+@ToString(callSuper = true)
+@EqualsAndHashCode(callSuper = true)
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class UpdatedUnbanRequest extends UnbanRequest {
+
+    private String resolverId;
+
+    private String resolverLogin;
+
+    private String resolverMessage;
+
+    private String status; // e.g. "CANCELED" or "APPROVED" or "DENIED"
```

Comment:
I think normally a enum would be a better choice here as we only have 3 possible values, but since this topic is not official / undocumented a string is fine too as it's more flexible in case they introduce new / change status codes.

Reply:
Agreed, that was my primary reasoning for leaving it as a string